doc/api/webcrypto.md

```diff
@@ -575,20 +575,30 @@ changes:
 
 * `algorithm`: {AlgorithmIdentifier|EcdhKeyDeriveParams|HkdfParams|Pbkdf2Params}
 * `baseKey`: {CryptoKey}
-* `length`: {number}
+* `length`: {number|null}
 * Returns: {Promise} containing {ArrayBuffer}
 
 <!--lint enable maximum-line-length remark-lint-->
 
 Using the method and parameters specified in `algorithm` and the keying
 material provided by `baseKey`, `subtle.deriveBits()` attempts to generate
-`length` bits. The Node.js implementation requires that `length` is a
-multiple of `8`. If successful, the returned promise will be resolved with
-an {ArrayBuffer} containing the generated data.
+`length` bits.
+
+The Node.js implementation requires that when `length` is a
+number it must be multiple of `8`.
+
+When `length` is `null` the maximum number
+of bits for a given algorithm is generated. This is allowed for the
+`'ECDH'`, `'X25519'`[^1], and `'X448'`[^1] algorithms.
```

Comment:
Nit: I personally prefer re-formatting when it does not increase the diff size so that the markdown document remains readable:

```suggestion
When `length` is `null` the maximum number of bits for a given algorithm is
generated. This is allowed for the `'ECDH'`, `'X25519'`[^1], and `'X448'`[^1]
algorithms.
```